### ISSUE TITLE
Annotation ??!

### ISSUE THREAD

**Issue by sinamiandashti** [EvanDotPro/EdpSuperluminal#12]
actually your module not works with classes that have annotation

because it doesn't minify the doc block above the variable ... just ignoring them

or your module does not support comments !

**Comment by EvanDotPro** [comment]
This is true -- we'd have to either figure out a way to omit files with annotations or include all comments or just comments we detect that are annotations. Omitting them would probably be the quickest solution.

**Comment by bacinsky** [comment]
I think comment annotations are just a shortcut for configuration, so (if any) annotations should be cached, like an ordinary config.php file. Preserving annotations is not good, because than PHP files can't be minified without comments.

**Comment by seyfer** [comment]
Doctrine doesn't work without annotations!

**Comment by lsklyut** [comment]
@bacinsky I'm interested in what you mean by caching them like an ordinary config.php file - could you please elaborate?

EDIT: I forgot that the module does not cache non-Zend classes; @seyfer could you please elaborate on how Doctrine is broken by this module?

I think we could keep an array of known classes/namespaces which break this cache, and simply ignore them ( #21 and I believe #28 for Zend\Db )
